frontend/interfaces/package_type.ts

```diff
@@ -0,0 +1,22 @@
+const unixPackageTypes = ["pkg", "deb"] as const;
+const windowsPackageTypes = ["msi", "exe"] as const;
+export const packageTypes = [
+  ...unixPackageTypes,
+  ...windowsPackageTypes,
+] as const;
+
```

Comment:
Nice! Just a heads up next sprint, we will likely be adding .dmg (mac), .zip (cross platform), and .rpm (linux)